metric_learn/itml.py

```diff
@@ -176,7 +188,9 @@ def fit(self, pairs, y, bounds=None):
     self : object
         Returns the instance.
     """
-    return self._fit(pairs, y, bounds=bounds)
+    self._fit(pairs, y, bounds=bounds)
+    self.threshold_ = np.mean(self.bounds_)
```

Comment:
to remove

Reply:
I guess if the above is yes you mean remove this and replace it by `calibrate_threshold` like in all pairwise metric learners ?